nylas/client/restful_models.py

```diff
@@ -781,7 +791,7 @@ def __init__(self, api):
         NylasAPIObject.__init__(self, Account, api)
 
     def as_json(self):
-        dct = NylasAPIObject.as_json(self)
+        dct = {"metadata": self.metadata}
```

Comment:
are the other fields from an account not needed anymore?

Reply:
There's nothing you can PUT to an account other than `metadata`.